src/components/ActionElement/ActionWrapper.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+MIT License
+
+Copyright (c) 2021-present, Elastic NV
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+import { EuiFlexItem } from "@elastic/eui";
+import React from "react";
+import styled from "styled-components";
+
+const AssertItem = styled(EuiFlexItem)`
+  border-left: ${props => props.theme.border.thick};
+  padding-left: 20px;
+  margin-left: 50px;
+`;
+
+const ActionItem = styled(EuiFlexItem)`
+  border-left: ${props => props.theme.border.thick};
+  padding-left: 20px;
+  margin-left: 0px;
+`;
+
+interface IActionWrapper {
+  isAssert?: boolean;
+}
+
+export const ActionWrapper: React.FC<IActionWrapper> = ({
```

Comment:
Hm - I do think this is a conversation worth having. You may have noticed that up until this PR, the entity we refer to colloquially as `Steps` was an `ActionContext[][]` throughout the codebase.

We have always referred to `ActionContext` as `Action`, even though it contains a field like `{ action: Action }`. Then we have `Assertion`, which we are basically saying is a sub-type of `ActionContext`. We should probably have a separate breakout to nail down what we want to call all of these, and do a follow-up PR where we rename the types to correspond to the formalized definitions.

As I mentioned above, in this particular case, I called this `ActionWrapper` because we think of `Action` and `Assertion` both as actions. I'm happy to rename this to `Behavior`.